requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+pytest
+pyinstaller
```

Comment:
Might be good to include versions so we dont get caught out by updates to these libraries

Reply:
These are left specifically to the latest version, since they're not part of the shipped library.

Leaving them to the latest allows for a speed increase on GH runners, because these are then taken out of the GH pypi cache, instead of downloaded everytime.